### ISSUE TITLE
Circular paths

### ISSUE THREAD

**Issue by DanCuber** [cabaletta/baritone#702]
Add the ability to set circular paths, with the ability to select the radius, centerpoint and y height of it.

Usage/command syntax:
"#circle <center> <radius> <y height (omit to ignore height)>"
or
"#circlepath" instead of "#circle"

**Comment by ZeroMemes** [comment]
I think a very good question is ***W H Y***

**Comment by StijnSimons** [comment]
Real men walk circles... duh...

**Comment by IronException** [comment]
actually I had a usecase for this when I made this spawn render: https://www.reddit.com/r/2b2t/comments/ccgffq/render_of_spawn_20190710_society_project/?utm_medium=android_app&utm_source=share.
But I cant find any other. Also doing this isnt that easy. The only algorythm that comes to my mind right now requires a parameter how accyrate the circle has to be.

**Comment by ZeroMemes** [comment]
Yeah, it would require making a ton of small segments per an accuracy parameter or some calculated amount based on size. But that isn’t too difficult.
